tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -25,6 +32,15 @@ export interface ISnippetConfig {
     cr?: boolean; // cdn retry would be proceed if ture
     dle?: boolean; // Custom optional value to disable sdk load error to be sent
     sri?: boolean; // Custom optional value to specify whether fetching the snippet from integrity file and do integrity check
+    pl?: boolean; // Custom optional value to specify whether to enable the trusted type policy check on snippet
+    /**
+     * Custom optional value to specify the name of the trusted type policy that would be implemented on the snippet, default is '1ds-default'
```

Comment:
As this is currently only the "Application Insights" snippet (it doesn't really work for 1ds today) the default should be `aiPolicy` as on line 33